plugins/reach/lib/model/enums/VendorServiceTurnAroundTime.php

```diff
@@ -7,16 +7,16 @@ interface VendorServiceTurnAroundTime extends BaseEnum
 {
 	const BEST_EFFORT           = -1;
 	const IMMEDIATE             = 0;
+	const ONE_BUSINESS_DAY      = 1;
```

Comment:
Since the numeric value was number of seconds so far, in case of Business days better use value that cannot be used otherwise - '1B' , '2B' etc....